guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -23,9 +23,7 @@ endif::[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-ifndef::foreman-deb[]
-# dnf install fapolicyd
-endif::[]
+# {project-install-package} fapolicyd
```

Comment:
Here, it is `satellite-maintain packages install fapolicyd` which I guess, is incorrect. 
It should be `dnf install fapolicyd` because this package is the part of BaseOS repo. 

So, can we use `:client-install-package-el8:` ?